README.md

```diff
@@ -1,16 +1,148 @@
 # OpenCensus Specs
 
-## General Design
-* For details about the library structured see [Namespace and Package](https://github.com/census-instrumentation/opencensus-specs/blob/master/NamespaceAndPackage.md)
+This is a high level design of the OpenCensus library, some of the API examples may be written 
+(linked) in C++, Java or Go but every language should translate/modify them based on language 
+specific patterns/idioms. Our goal is that all libraries have a consistent "look and feel".
 
-## Trace Design
-* Trace API is described [here](https://github.com/census-instrumentation/opencensus-specs/blob/master/trace/README.md)
+This repository uses terminology (MUST, SHOULD, etc) from [RFC 2119][RFC2119].
 
-* Data model is defined [here](https://github.com/census-instrumentation/opencensus-proto/blob/master/trace/trace.proto)
+## Overview
+Today, distributed tracing systems and stats collection tend to use unique protocols and 
+specifications for propagating context and sending data to backend processing systems. This 
```

Comment:
add a word "sending diagnostic data" Otherwise, your system will be hijacked for data provenance or other workflow tracking reasons :)

Reply:
Done.